layers/poi/layer.sql

```diff
@@ -36,9 +36,14 @@ RETURNS TABLE(osm_id bigint, global_id text, geometry geometry, name text, name_
                 THEN NULLIF(information, '')
             WHEN subclass = 'place_of_worship'
                     THEN NULLIF(religion, '')
+            WHEN subclass = 'pitch'
+                    THEN NULLIF(sport, '')
```

Comment:
These subclass rules are hidden in the tile query, and can't be retrieved by Fafnir. (And as a consequence icons used on the map will not match on the UI).  
Should we define a new function poi_subclass, similar to `poi_class` ? (Not so convincing: the signature of this function would change when new tags are used...).

As far as i can see, this is the only remaining problem (but not directly related to this PR).